daemons/scheduled-ci-build/vendor.in

```diff
@@ -0,0 +1 @@
+../../vendor.in
```

Comment:
I don't think it depends on the vendored packages.

Reply:
Since my 848a7e0, app.py imports `dss` to configure logging. Until now I didn't realize that I had broken this lambda. With the original minimal requirements.txt I got

```
Unable to import module 'app': No module named 'connexion'
```

Symlinking in the top-level `requirements.txt` gave

```
Unable to import module 'app': No module named 'yaml'
```

… and symlinking vendor.in fixed that. This could be whittled down to some extend but for this purpose I think it is acceptable to have standardized logging at the expense of a bloated dependency tree.